tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+import re
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import generate_params, regular_file_cud, LOG_FILE_PATH, callback_num_inotify_watches, \
+                              detect_initial_scan, CHECK_ALL, REQUIRED_ATTRIBUTES
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.darwin, pytest.mark.sunos5, pytest.mark.tier(level=0)]
+
+# variables
+
+
+test_directories = [os.path.join(PREFIX, 'dir')]
+
+directory_str = str(test_directories[0])
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_check_realtime.yaml')
+testdir = test_directories
+test_file = "testfile.txt"
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# configurations
+
+
+conf_params = {'TEST_DIRECTORIES': directory_str, 'MODULE_NAME': __name__}
+parameters, metadata = generate_params(extra_params=conf_params, modes=['scheduled'])
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+# helper functions
+
+
+def callback_ignore_realtime_flag(line):
+    match = re.match(r".*Ignoring flag for real time monitoring on directory: (.+)$", line)
+    if match:
+        return True
+
+# fixtures
+
+
+@pytest.fixture(scope='function', params=configurations)
+def check_realtime_mode_failure():
+
+    wazuh_log_monitor.start(timeout=60, callback=callback_ignore_realtime_flag,
+                            error_message='Did not receive expected "Ignoring flag for real time monitoring on  \
+                            directory: ..." event', update_position=False)
+    detect_initial_scan(wazuh_log_monitor)
+
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.mark.parametrize('folder', testdir)
+@pytest.mark.parametrize('file', [test_file])
+# tests
```

Comment:
```suggestion
# tests
@pytest.mark.parametrize('folder', testdir)
@pytest.mark.parametrize('file', [test_file])
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/1ff97d48c736c3eececd80e24a06b8e6de298b9c)